vlib/v/checker/tests/noreturn_with_return.out

```diff
@@ -0,0 +1,13 @@
+vlib/v/checker/tests/noreturn_with_return.vv:2:1: error: [noreturn] functions can not use return statements
```

Comment:
`functions cannot use return statements`

Reply:
only [noreturn] functions can not return, normal functions can

Reply:
i mean: `can not` to `cannot` to be consistent with other errors.

Reply:
oh, you are right